change/@microsoft-teams-js-f9ac3559-b7e4-4cdd-bff9-54aecd663d48.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Made `externalAppAuthenticationForCEA.ts` throw `Error` objects instead of `SdkErrors`",
+  "packageName": "@microsoft/teams-js",
+  "email": "trharris@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
Should this part still be patch?

Reply:
I believe this is correct, yes, it's just saying what a package that depends on this should bump it's version number by when it consumes a new version.

The funny (?) part is that I think this isn't really necessary because patch should be the default here? Never hurts to be specific I guess.